### PR TITLE
Resprites mechs, adds a few new things

### PR DESCRIPTION
**Pull request by kyres1**
Fully resprites mech parts and equipment from top to bottom with very little exception. Redoes the balance of a few parts to make them more unique as well, and also just outright adds a superheavy exosuit in place of the Marauder once seen in the HAPT shuttle. Full layout and changes are below.

layout

Adds light thrusters, extremely fragile but faster than heavy thrusters.

Replaces hoverthrusters with heavy thrusters, bulky and more durable versions that go much slower.

Adds a superheavy mech type that is just a really buff heavy mech (seen at the end of the layout.)

Redescribes a few things to make their purpose, advantages and disadvantages more clear ICly.

Rebalances the combat mech parts entirely - formerly they had pretty crazy stats, with some stats being really low (10 hp legs???) and other stats being too high for the benefits, such as its x-ray vision that nobody knew they had.  Also restricts these parts from appearing on random mechs in cargo. 

Finally, added industrial spiderlegs - basically just spiderlegs, but slower with higher power draw in exchange for more armor and a cool new sprite.

Finally but really this time, adds a trait to mecha equipment that permits the icon states to use "_far" at the end, permitting long parts to be seen while on the opposing side of the viewer (for example, viewing right-hand sprites while the sprite faces west.)

By request of ferner, also removes some of the code permitting visible pilots seeing as we have no plans to utilize it now, or in the future. Other codework that wasn't simply copy/pasting may well be their doing too, as they helped me with this PR immensely.

will do changelog soon

### CODE REVIEW

File: code/modules/heavy_vehicle/mech_interaction.dm [hunk 1]
```diff
@@ -233,14 +233,11 @@
...
 	LAZYDISTINCTADD(pilots, user)
 	sync_access()
-	update_pilot_overlay()
 	playsound(src, 'sound/machines/windowdoor.ogg', 50, 1)
 	user << sound('sound/mecha/nominal.ogg',volume=50)
 	if(user.client) user.client.screen |= hud_elements
 	LAZYDISTINCTADD(user.additional_vision_handlers, src)
 	update_icon()
-	update_pilot_overlay()
-	return 1
```
Comment (fernerr) [inline code/modules/heavy_vehicle/mech_interaction.dm:243]: Return the return

File: code/modules/heavy_vehicle/mech_interaction.dm [hunk 2]
```diff
@@ -275,8 +272,6 @@
 		set_intent(I_HURT)
 		LAZYREMOVE(pilots, user)
 		UNSETEMPTY(pilots)
-		update_pilot_overlay()
-	return
```
Comment (fernerr) [inline code/modules/heavy_vehicle/mech_interaction.dm:279]: Return the return

Comment (NonQueueingMatt) [reply code/modules/heavy_vehicle/mech_interaction.dm:279]: That return doesn't actually seem to do anything from what I can see

Comment (fernerr) [reply code/modules/heavy_vehicle/mech_interaction.dm:279]: The proc was removed by my instruction with the returns being mistakenly removed, but if this return does nothing then its removal is fine.

**Comment by kyres1** [comment]
!review

**Review by Alberyk — APPROVED** [review]

**Review by skull132 — APPROVED** [review]

**Pull request merged**